### PR TITLE
fix migrate for ssb-db@19.3.1

### PR DESCRIPTION
**Pull request by staltz**
Manyverse uses ssb-db@19.3.1 and the `sizeStream` in migrate wasn't working, while ssb-db2 uses ssb-db@20.3.0 for the tests.

I changed ssb-db to 19.3.1 in this module's package.json, then tests didn't pass either. Then I fixed the implementation until tests passed for 19.3.1. And then I changed package.json back to 20.3.0. Unfortunately I don't know a way of installing both versions and running tests, so we just have to remember to manually test 19.3.1 if we change `migrate`'s implementation. But now we know `migrate` works for both versions.

### CODE REVIEW

File: migrate.js [hunk 1]
```diff
@@ -32,29 +32,41 @@ function makeFileExistsObv(filename) {
...
+      raw: true,
+      old: true,
+      live: false,
+    })
+    const logStreamLive = sbot.createLogStream({
+      raw: true,
+      old: false,
+      live: true,
+    })
+    const getSize = () => sbot.status().sync.since
```
Comment (arj03) [inline migrate.js:52]: Should this be since.value instead?

Comment (arj03) [reply migrate.js:52]: There same a few lines up

**Comment by arj03** [comment]
Just a small comments, otherwise looks good.

File: migrate.js (hunk 1 above)
Comment (staltz) [reply migrate.js:52]: It's weird, isn't it? But it's correct,

- see 20.3.0: https://github.com/ssbc/ssb-db/blob/da89b7552e054c9e8baaad78506714e13a62a7ec/index.js#L107
- see 19.3.1: https://github.com/ssbc/ssb-db/blob/6c0cdd580b6166b106283e3f1745585e189252ce/index.js#L94

Comment (arj03) [reply migrate.js:52]: OMG!

**Comment by arj03** [comment]
Looks good :-)

**Pull request merged**